src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+/**
+ * Controls application.
+ */
+
+public class Duke {
+    /** Local task store */
+    private final Storage storage;
+    /** List of tasks */
+    private TaskList tasks;
```

Comment:
Maybe list instead of tasks would be clearer?

Reply:
I'm worried that it may not be clear that it's referencing a list of tasks if it were just named list - what about taskList?

Reply:
Sure that sounds good!